docs/tutorials/understanding-datamodels.rst

```diff
@@ -304,8 +320,12 @@ Note, that the left-hand-side of the entry point,
     the ``mappings``, ``v5`` and ``v6`` folders, in which case the entry points
     won't be correctly discovered.
 
+External representation
+-----------------------
+As it was mentioned before there was an external and an internal representation. This is how the data will be parsed from user input and how it will be shown back to them. As it can be seen, both serializers and loaders depend on ``Marshmallow`` schemas. It is not mandatory to use it but it is **highly advised**. However, you can create your own if the given ones do not fit your data model.
```

Comment:
I would use present tense. `there is an external and an internal`.

`However, you can create your own schemas ...`